poker-pals/src/components/Poker.vue

```diff
@@ -145,27 +147,27 @@ export default {
                     player.cards = {card1: {src: null}, card2: {src: null}};
                     player.bet = 0;
                     player.accountName = "";
-                    player.accountImage.src = this.imageFiles.find(file => file.name === 'invisible').src;
+                    player.accountImage.src = null;
                     player.chipTotal = "";
                     player.action = "WAITING";
                     player.youTag = false;
                     player.timer = false;
                 }
-                this.$router.push({ name: "Tables", params: {authenticated: true, userID: this.userID} });
+                this.$router.push({ name: "Tables", params: {authenticated: true, socket: this.socket, userID: this.userID} });
             });
 
             this.socket.on('beginTheGame', msgJSON => { // each player receives their two personal cards upon the game starting
                 let msg = JSON.parse(msgJSON);
-                this.myCards[0].src = this.imageFiles.find(file => file.name === msg[0]).src;
-                this.myCards[1].src = this.imageFiles.find(file => file.name === msg[1]).src;
+                this.myCards[0].src = this.imageFiles.getImage(msg[0]).src;
+                this.myCards[1].src = this.imageFiles.getImage(msg[1]).src;
```

Comment:
are you receiving an array of the card image names here? If this is all the data in the msgJSON, could you rename msg so that it's clearly related to images?

Reply:
done